src/snowflake/snowpark/modin/plugin/utils/numpy_to_pandas.py

```diff
@@ -112,6 +112,36 @@ def may_share_memory_mapper(a: Any, b: Any, max_work: Optional[int] = None) -> b
     return False
 
 
+def full_like_mapper(
+    a: Union[pd.DataFrame, pd.Series],
+    fill_value: Hashable,
+    dtype: Optional[Any] = None,
+    order: Optional[str] = "K",
+    subok: Optional[bool] = True,
+    shape: Optional[tuple[Any]] = None,
+) -> Union[pd.DataFrame, pd.Series]:
+    if not subok:
+        return NotImplemented
+    if not order == "K":
+        return NotImplemented
+    if dtype is not None:
```

Comment:
Why can't we support dtype here? It just overrides the datatype provided by a right?

Reply:
These are numpy types; and within the context of mapping them to snowpandas they don't make sense and it would be overly complicated to support the options.